db/migrate/20150114172336_create_answers.rb

```diff
@@ -1,6 +1,9 @@
 class CreateAnswers < ActiveRecord::Migration
   def change
     create_table :answers do |t|
+      t.text :body
+      t.references :user, index: true
+      t.references :question, index: true
```

Comment:
there are 2 foreign keys, it's usually best to define a composite index for both, and a single index for the "second" one.

```
add_index :answers, [:user_id, :question_id]
add_index :answers, :question_id
```